uploader.json

```diff
@@ -3072,6 +3067,11 @@
             "symbolic_name": "kubepods_cpu_quota",
             "pattern": []
         },
+        {
+            "file": "/var/opt/rh/rh-mysql*/log/mysql/mysqld.log",
```

Comment:
@JoySnow `"file"` needs to change to `"glob"`.

Reply:
@bfahr  Definitely, thanks for point it out.